internal/pkg/aws/apprunner/service.go

```diff
@@ -33,6 +34,11 @@ type EnvironmentVariable struct {
 	Value string
 }
 
+type EnvironmentSecret struct {
```

Comment:
// EnvironmentSecret contains the name and value of a Secret from SSM Parameter Store or Secrets Manager.
Will add the above doc string in next commit. Missed in this commit.